### DISCUSSION TITLE
Is it possible to use/build Bootstrap with unique prefixes in classes?

### DISCUSSION THREAD

**Discussion by artalar** [twbs/bootstrap#33874] [Q&A]
Is it possible to use/build Bootstrap with unique prefixes in classes?

**Comment by ffoodd** [comment]
There's nothing in Bootstrap to handle this. This should be doable using PostCSS or anything that processes compiled CSS, but I never did that so cannot help you more.

**Comment by dtslvr** [comment]
This would be a great feature to avoid clashes with other style classes. Define something like `twbs-*` for `twbs-text-center` etc.
